### PR TITLE
Small design fixes

### PR DESCRIPTION
**Pull request by georgeschena**
screenshot 2019-01-17 at 15 54 02
screenshot 2019-01-17 at 15 53 46

### CODE REVIEW

**Comment by kjdchapman** [comment]
Would you mind adding screenshots? It's difficult to parse these sorts of changes without 'em, and checking out the branch myself means I won't be able to get to this as quickly.

File: app/assets/stylesheets/components/overview.scss [hunk 1]
```diff
@@ -13,3 +13,12 @@
 .summary-link-colour:hover {
   color: govuk-colour("white")
 }
+
+.overview-section-allignment{
```
Comment (kjdchapman) [inline app/assets/stylesheets/components/overview.scss]: nit: spelling

```suggestion
.overview-section-alignment{
```

File: app/assets/stylesheets/components/overview.scss [hunk 2]
```diff
@@ -13,3 +13,12 @@
...
+  padding-left: 0px;
+  padding-right: 0px;
+}
+
+.activity-section-allignment{
```
Comment (kjdchapman) [inline app/assets/stylesheets/components/overview.scss]: nit: spelling

```suggestion
.activity-section-alignment{
```

File: app/assets/stylesheets/components/team_members.scss [hunk 3]
```diff
@@ -10,3 +10,7 @@
     color: $govuk-link-hover-colour;
   }
 }
+
+.list-item-padding{
```
Comment (kjdchapman) [inline app/assets/stylesheets/components/team_members.scss]: nit: bracket spacing

```suggestion
.list-item-padding {
```

(maybe we need a SCSS linter)

File: app/assets/stylesheets/components/overview.scss [hunk 4]
```diff
@@ -13,3 +13,12 @@
...
+  margin-left: -15px;
```
Comment (kjdchapman) [inline app/assets/stylesheets/components/overview.scss]: I'd be surprised if this works on all screen sizes/devices. 

Is there any way for us to achieve this without resorting to pixel-based styling?

File: app/views/home/index.html.erb [hunk 5]
```diff
@@ -34,8 +34,8 @@
   </div>
   <div class="govuk-grid-column-full" id="user-statistics">
     <hr class="govuk-section-break--m govuk-section-break--visible">
-    <h2 class="govuk-heading-l govuk-grid-column-full">Activity</h2>
-    <div class="govuk-grid-column-full" id="unique-users">
+    <h2 class="govuk-heading-l govuk-grid-column-full  activity-section-allignment">Activity</h2>
```
Comment (kjdchapman) [inline app/views/home/index.html.erb]: nit: extra space
```suggestion
    <h2 class="govuk-heading-l govuk-grid-column-full activity-section-allignment">Activity</h2>
```

File: app/views/home/index.html.erb [hunk 6]
```diff
@@ -34,8 +34,8 @@
...
+    <div class="govuk-grid-column-full activity-section-allignment" id="unique-users">
```
Comment (kjdchapman) [inline app/views/home/index.html.erb]: nit: spelling

```suggestion
    <div class="govuk-grid-column-full activity-section-alignment" id="unique-users">
```

**Comment by kjdchapman** [comment]
Lots of nitpicking, but one _actual_ question around pixel-specific styling.

File: app/assets/stylesheets/components/overview.scss (hunk 4 above)
Comment (georgeschena) [reply app/assets/stylesheets/components/overview.scss]: It works on the multiple smaller displays, I tested it ensuring it was responsive

**Pull request merged**